great_expectations/datasource/fluent/pandas_dbfs_datasource.pyi

```diff
@@ -0,0 +1,284 @@
+import re
+import typing
+from logging import Logger
+from typing import TYPE_CHECKING, Hashable, Iterable, Optional, Sequence, Union
+
+from typing_extensions import Literal
+
+from great_expectations.core._docs_decorators import public_api as public_api
+from great_expectations.datasource.fluent import PandasFilesystemDatasource, Sorter
+from great_expectations.datasource.fluent.data_asset.data_connector import (
+    DBFSDataConnector as DBFSDataConnector,
+)
+from great_expectations.datasource.fluent.interfaces import (
+    SortersDefinition as SortersDefinition,
+)
+from great_expectations.datasource.fluent.interfaces import (
+    TestConnectionError as TestConnectionError,
+)
+
+if TYPE_CHECKING:
+    from great_expectations.datasource.fluent.dynamic_pandas import (
+        CompressionOptions,
+        CSVEngine,
+        FilePath,
+        IndexLabel,
+        StorageOptions,
+    )
+    from great_expectations.datasource.fluent.pandas_file_path_datasource import (
+        CSVAsset,
+        ExcelAsset,
+        FeatherAsset,
+        HDFAsset,
+        HTMLAsset,
+        JSONAsset,
+        ORCAsset,
+        ParquetAsset,
+        PickleAsset,
+        SASAsset,
+        SPSSAsset,
+        STATAAsset,
+        XMLAsset,
+    )
+
+logger: Logger
+
+class PandasDBFSDatasource(PandasFilesystemDatasource):
+    type: Literal["pandas_dbfs"]  # type: ignore[assignment]
```

Comment:
```suggestion
    type: Literal["pandas_dbfs"]  = "pandas_dbfs"
```
Does it go away if you do this?

Reply:
@Kilo59 I will try.  Bu just to clarify: this is the PYI file -- is setting a value appropriate?  Thanks.